bundles/framework/admin-announcements/view/AnnouncementsForm.jsx

```diff
@@ -0,0 +1,157 @@
+  const rangeInitial = () => {
+
+    if (form.begin_date && form.end_date) {
+      return [moment(form.begin_date, dateFormat), moment(form.end_date, dateFormat)]; 
+    } else {
+      return [moment(moment(),dateFormat), moment(moment(),dateFormat)];  
+    }
+
+  } 
+
+  //Active value set depending on if creating a new announcement or editing an old one
+  const activeInitial = () => {
+    if(form.active === undefined) {
+      return true;
+    } else {
+      return form.active;
+    }
+  }
+
+    return (
+      <div>
+            <Form layout="vertical" 
+              onFinish={onFinish} 
+              initialValues={{
+                title: form.title !== undefined ? form.title : title,
+                content: form.content,
+                range_picker: rangeInitial(),
+                active: activeInitial(),
+              }}>
+              <Form.Item
+                name="title"
+                label={<Message messageKey='newAnnouncement.title' />}
+                rules={[
+                  {
+                    required: true,
+                    message: <Message messageKey='titleError' />,
+                    whitespace: true
+                  }
+                ]}
+                value={title}
+                onChange={ evt => controller.updateTitle(evt.target.value)}
+              >
+                <Input />
+              </Form.Item>
+              <Form.Item
+                name="content"
+                label={<Message messageKey='newAnnouncement.content' />}
+                rules={[
+                  {
+                    required: true,
+                    message: <Message messageKey='contentError' />,
+                    whitespace: true
+                  }
+                ]}
+              >
+                <TextArea rows={4}/>
+              </Form.Item>
+              <Form.Item
+                name="range_picker"
+                label={<Message messageKey='newAnnouncement.date-range' />}
+                {...rangeConfig}
+              >
+                <DateRange popupStyle={{zIndex: '999999'}} />
+              </Form.Item>
+              <Form.Item name="active" label={<Message messageKey='newAnnouncement.show-popup' />} valuePropName="checked">
+                <Switch onClick={() => controller.toggleActive()}/>
+              </Form.Item>
+              <Row>
+                <Form.Item>
+                  <Button type="primary" htmlType="submit" >
+                    <Message messageKey={'save'}/>
+                  </Button>
+                </Form.Item>
+                <Form.Item>
+                  <Confirm
+                      title={<Message messageKey='messages.deleteAnnouncementConfirm'/>}
+                      onConfirm={() => controller.deleteAnnouncement(index, form.id, form.title)}
+                      okText={getMessage('yes')}
+                      cancelText={getMessage('cancel')}
```

Comment:
The getMessage() returns Message-tag. You use the Message-tag for title, why use the function wrapper here?